README.md

```diff
@@ -322,7 +322,7 @@ If you don't know all the attributes that you want to place in a component ahead
 
 ```jsx
 const TodoList = ({ todos, toggle }) =>
-  todos.map(todo => <TodoItem {...todo} toggle={toggle} />)
+  (todos.map(todo => <TodoItem {...todo} toggle={toggle} />)
```

Comment:
wasn't the parentheses matching the one on the `.map`, I don't think it was wrong before.

Reply:
Thanks, @fuzetsu, you are right. My bad for not thoroughly checking @ricosmall's PR in detail.

Fixed in #775. 👋